src/validateEmail.test.js

```diff
@@ -8,13 +8,30 @@ describe(`Function 'validateEmail':`, () => {
   });
 
   it(`should return boolean`, () => {
+    expect(typeof validateEmail('goodEmail@.com')).toBe('boolean');
+  });
 
+  it(`should return 'true' for the valid email`, () => {
+    expect(validateEmail('test838@gmail.com.')).toBeTruthy();
+    expect(validateEmail('userName.userSurname@gmail.com.')).toBeTruthy();
+    expect(validateEmail('anabella123Syphora@g.com.')).toBeTruthy();
+    expect(validateEmail('SirkoIvan@uk.com')).toBeTruthy();
```

Comment:
The test cases for valid emails include trailing dots (e.g., 'test838@gmail.com.'). Typically, valid email addresses should not end with a dot. Please verify if this is the intended behavior of the `validateEmail` function.

Reply:
should be intended